cfg.yaml

```diff
@@ -38,9 +42,13 @@ templates:
 
 # Outputs are target services that should consume the messages
 outputs:
+- name: stdout
+  type: stdout
+  enable: true
+
 - name: my-jira   #  name must be unique
   type: jira      #  supported types: jira, email
-  enable: true
```

Comment:
let's keep them enabled

Reply:
I've set enable fields to `false` because it's a part of output settings.